baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -372,14 +373,31 @@ export class AddressesClient {
       'project': request.project || '',
     });
     this.initialize();
-    return this.innerApiCalls.delete(request, options, callback);
+    return new Promise<[
```

Comment:
Why do you need to create a new Promise here? Can't we just do `return this.innerApiCalls.delete(...).then(...)`?  Also, there will be no need to `.catch` anything since the error will be just thrown and the promise will get rejected.